src/router/routes.js

```diff
@@ -40,6 +40,11 @@ const routes = [
       }
     ]
   },
+  {
+    path: '/why',
+    name: 'Why',
```

Comment:
We are not using the `name` property on routes in this project. Route pushes are being done directly with the route name.